.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ executors:
     parameters:
       image:
         type: string
-        default: "14.17.0"
+        default: "14.18.1"
```

Comment:
We should kee pit as close as possbile to 14.15.0 IMO, I think 14.17 was picked because there wasn't a lower version available

Reply:
You're sure? 
![image](https://user-images.githubusercontent.com/16143594/140905291-65db1506-6596-4b0e-b677-9b02a824913e.png)

https://circleci.com/developer/images/image/cimg/node

My thinking behind this was that people probably use `nvm install 14` or even `nvm install 16` so they won't use 14.15.0 anyways. But on the other hand, we wanna make sure it works on that.

So yeah, maybe for that we should use 14.15.0 instead of 14.17.0

Reply:
Done!